### PR TITLE
Added checklist and simplified documentation

### PR DESCRIPTION
**Pull request by jakobgrunig**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,69 +1,24 @@
-# MDTP Contributor Guidelines
+# Contributor Guidelines
 
-Hello! Thank you for taking the time to contribute to the [HMRC Multichannel Digital Tax Platform](https://hmrc.github.io) (MDTP).
+Hello! If you are reading this, it is probably because you want send emails.
```
Comment (thoeni) [inline CONTRIBUTING.md]: *to send

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -78,40 +33,90 @@ One of our priorities is to make sure that the content of our emails doesn't lea
...
+```
+
+### I have created a pull request - now what?
+
+Once you've got your code changes done and a pull request created, it's time to get it pushed to production. Here's how:
+
+1. Create a story for project "Digital Contact (DC)" in Jira with details of the pull request and what it's for
+2. Tag our scrum-master [~chris.selby] in the ticket, so we can get it prioritised for you
+3. Join Slack channel #team-digital-contact to ask about progress and/or ask questions
+4. Our QA's will contact you when the they have merged and deployed to QA so you can check your happy before we create a release ticket
```
Comment (thoeni) [inline CONTRIBUTING.md]: you're

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,69 +1,24 @@
...
 
-It is the responsibility of both the Pull Requestor and Owning Team to follow these guidelines. If the Pull Requestor does not follow these guidelines, the Owning Team will reject the changes and may resist future Pull Requests. 
+## How to submit new or changed email template - Checklist 
 
-> It is important to remember that an Issue and a Pull Request are a placeholder for a conversation between the Pull Requestor and the Owning Team - people and interactions are important than processes and tools. The Pull Requestor should contact the Owning Team as soon as changes to the Repository are considered.
+1. Write the content for your email. One of your priorities should be to make sure that the email is not marked as spam by the user. See our [Content recommendations](#content-recommendations)
+2. Decide what is the most suitable priority for the template. See our [Volumes and priority recommendations](#volumes-and-priority-recommendations)
+3. Fork this repository and make the necessary code changes. See [How to add a template](#how-to-add-a-template)
+4. Verify that your template looks right when rendered - use the [preview mode](/README.md#preview-development)
```
Comment (thoeni) [inline CONTRIBUTING.md]: `[preview mode](/README.md#preview-mode)`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,69 +1,24 @@
...
 
-It is the responsibility of both the Pull Requestor and Owning Team to follow these guidelines. If the Pull Requestor does not follow these guidelines, the Owning Team will reject the changes and may resist future Pull Requests. 
+## How to submit new or changed email template - Checklist 
 
-> It is important to remember that an Issue and a Pull Request are a placeholder for a conversation between the Pull Requestor and the Owning Team - people and interactions are important than processes and tools. The Pull Requestor should contact the Owning Team as soon as changes to the Repository are considered.
+1. Write the content for your email. One of your priorities should be to make sure that the email is not marked as spam by the user. See our [Content recommendations](#content-recommendations)
+2. Decide what is the most suitable priority for the template. See our [Volumes and priority recommendations](#volumes-and-priority-recommendations)
+3. Fork this repository and make the necessary code changes. See [How to add a template](#how-to-add-a-template)
```
Comment (stjasink-ee) [inline CONTRIBUTING.md]: It might be worth saying "branch" instead of "fork" here to stick with git terminology and avoid confusion.

Comment (thoeni) [inline CONTRIBUTING.md]: I think in this case fork is appropriate as other teams don't have access (I guess) to branching our repo, they can just fork it and raise PRs from there. But I might be wrong...

Comment (stjasink-ee) [inline CONTRIBUTING.md]: If that is the usual process then ignore my comment :)

**Pull request merged**